airflow/providers/snowflake/transfers/s3_to_snowflake.py

```diff
@@ -71,6 +71,8 @@ class S3ToSnowflakeOperator(BaseOperator):
     :type session_parameters: dict
     """
 
+    template_fields = ("s3_keys",)
```

Comment:
```suggestion
    template_fields = ("s3_keys",)
    template_fields_renderers = {"s3_keys": "json"}
```
In this way the list will be rendered in more user friendly way. Related https://github.com/apache/airflow/issues/11177

Reply:
Cool ! Did not know that, thank you @turbaszek